io.github.pieterdd.StretchBreak.yaml

```diff
@@ -0,0 +1,43 @@
+id: io.github.pieterdd.StretchBreak
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: stretch-break
+finish-args:
+- "--share=ipc"
+- "--share=network"
```

Comment:
what is network for?

Reply:
Must've slipped in when copy/pasting from another manifest. I'll remove it.